src/azure/apim/contracts.ts

```diff
@@ -21,3 +21,12 @@ export interface IGatewayApiContract {
     id: string;
     name: string;
 }
+
+export interface IGatewayToken {
+    value: string;
+}
+
+export interface IGatewayTokenList {
```

Comment:
This is the GatewayKeyList.. Proper way to do this is to make a backend call and get the list of keys. You had this logic in your previous iteration to list keys